tox.ini

```diff
@@ -42,7 +42,7 @@ deps =
 
     dj111: Django>=1.11b1,<2.0
     dj111-mssql: django-pyodbc-azure==1.11.0.0
-    dj20: Django>=2.0,<2.1
+    dj20: Django>=2.0,<2.2
```

Comment:
instead of changes this you should use dj21: Django>=2.1rc1,<2.2

Reply:
ack; done this.